projects/power_distribution/src/smoke_uv_cutoff.c

```diff
@@ -0,0 +1,46 @@
+#include "smoke_uv_cutoff.h"
+
+#include "can_transmit.h"
+#include "front_uv_detector.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "pd_events.h"
+#include "pin_defs.h"
+#include "status.h"
+#include "wait.h"
```

Comment:
Once you've made the changes, remove any #includes that you don't use - the general rule is that you should #include a header if and only if you use some symbol declared in it in the file (like a function name, typedef, struct name, macro, global variable, enum value, etc).